spec/factories/trainees.rb

```diff
@@ -44,6 +44,51 @@
       add_attribute("date_of_birth(1i)") { form_dob.year.to_s }
     end
 
+    trait :not_started do
+      trainee_id { nil }
+      first_names { nil }
+      middle_names { nil }
+      last_name { nil }
+      gender { nil }
+      date_of_birth { nil }
+
+      diversity_disclosure { nil }
+      ethnic_group { nil }
+      ethnic_background { nil }
+      additional_ethnic_background { nil }
+      disability_disclosure { nil }
+
+      address_line_one { nil }
+      address_line_two { nil }
+      town_city { nil }
+      postcode { nil }
+      international_address { nil }
+      locale_code { nil }
+      email { nil }
+      commencement_date { nil }
+    end
+
+    trait :in_progress do
+      with_programme_details
+      with_start_date
+      degrees { [build(:degree, :uk_degree_with_details)] }
+    end
+
+    trait :completed do
```

Comment:
Ah I think I also added a similar trait `ready_for_trn_submission` - but yours are neater. Please keep your definitions when you do the final merge.